app/scripts/modules/core/src/cancelModal/CancelModal.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import { Modal } from 'react-bootstrap';
+import { Field, FieldProps, Form, Formik } from 'formik';
+import { IPromise } from 'angular';
+
+import { SubmitButton, ModalClose } from 'core';
```

Comment:
fyi I think or cargo cult says to prefer to import from the submodule of core, i.e.,  `core/modal`